docs/release/role-handbooks/README.md

```diff
@@ -0,0 +1,22 @@
+# role-handbooks
+
+These handbooks are maintained by current and previous contributors who have staffed these roles. They are intended to be living documents that evolve as the roles and project evolves. Do not treat them as rules set in stone, but guidelines to be re-examined.
+
+## Overview
+
+As a member of the Release Team, you play a crucial role in helping to ensure project releases
+are delivered smoothly and in a timely manner. While there are specific responsibilities for each sub-team within
+the Release Team, below are some general notes that every member of the Release Team might benefit by going
+through at the beginning of the cycle:
+
+- Slack:
+  - Most discussions related CAPI Release topics happens in the #cluster-api channel on the Kubernetes Slack. If you need access to the Kubernetes Slack, please visit <http://slack.k8s.io/>.
+- Kubernetes SIG membership:
+  - Try to become an official member of the Kubernetes SIG, if possible. More information on the membership and requirements can be found [here](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-team.md#cluster-api-release-team-vs-kuberneteskubernetes-sig-membership).
+- Familiarize yourself with the Release Process:
+  - Review the release tasks which explains the responsibilities and tasks for each role within the release team.
+    - [Release Lead Team](../role-handbooks/release-lead/README.md)
+    - [Comms Team](../role-handbooks/communications/README.md)
+    - [CI Signal Team](../role-handbooks/ci-signal-bug-triage-automation/README.md)
+- Check the Release Timeline:
+  - Go through the [release timeline](../releases/) of the release cycle you are involved in (i.e checkout `release-1.6.md` if you are part of the 1.6 cycle release team) to better understand the key milestones and deadlines.
```

Comment:
```suggestion
  - Go through the [release timeline](../releases/) of the release cycle you are involved in (i.e reference `release-1.6.md` if you are part of the 1.6 cycle release team) to better understand the key milestones and deadlines.
```
nit: in case users are confusing the wording with checking out a branch